web/src/main/java/org/openmrs/web/controller/LoginController.java

```diff
@@ -18,6 +18,8 @@
 import org.springframework.web.context.request.WebRequest;
 import org.springframework.web.util.JavaScriptUtils;
 
+import static org.openmrs.web.WebConstants.*;
```

Comment:
We discourage using wild card imports https://wiki.openmrs.org/display/docs/Coding+Conventions